cypress/integration/ete/sign_out/sign_out.3.cy.ts

```diff
@@ -8,6 +8,7 @@ describe('Sign out flow', () => {
 
   context('Signs a user out', () => {
     it('Removes IDAPI log in cookies and dotcom cookies when signing out', () => {
+      Cypress.Cookies.debug(true);
```

Comment:
Should this be here?

Reply:
good spot! i was using this for debugging the test